docs/usage/self-hosted-configuration.md

```diff
@@ -656,6 +656,10 @@ CI jobs are usually triggered by these events: pull-request creation, pull-reque
 Set as an integer.
 Default is no limit.
 
+## presetCache
+
+This option is to turn on caching for loading presets. By turning it on, loaded presets will be cached in `preset` namespace in package cache. TTL is 15 minutes by default, and it is adjustable in [cacheTtlOverride](#cachettloverride).
```

Comment:
```suggestion
When this feature is enabled, resolved presets will be cached in Renovate's package cache, enabling reuse across multiple repositories.
Doing so improves efficiency because shared presets don't need to be reloaded/resolved for every repository, however it also means that private presets can be "leaked" between repositories.
You should only enable this when all repositories are trusted, such as a corporate environment.

TTL is 15 minutes by default, and it is adjustable in [cacheTtlOverride](#cachettloverride).
```

Needs longer description of how to adjust in cacheTtlOverride. I would also recommend a longer TTL such as 30 or 60.